### PR TITLE
fix: add "trusted" to templates & commands

### PR DESCRIPTION
**Pull request by DekusDenial**
## Context

Add visual aid to `trusted` templates and commands

## Objective

prepend the svg for `trusted` to the name of template/command
and fix search on lowercase match

Screen Shot 2019-06-20 at 3 33 10 PM
Screen Shot 2019-06-20 at 3 33 23 PM

## References

https://github.com/screwdriver-cd/screwdriver/issues/1427

blocked by: 
https://github.com/screwdriver-cd/data-schema/pull/340
https://github.com/screwdriver-cd/datastore-sequelize/pull/52

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/commands/detail/controller.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ export default Controller.extend({
       return this.commands[0];
     }
   }),
+  trusted: computed('commands.[]', function computeTrusted() {
+    return this.commands.reduce((trusted, c) => trusted || !!c.trusted, false);
```
Comment (adong) [inline app/commands/detail/controller.js]: Does this line of code means if all commands are `trusted`, then mark as `trusted`? 

Do you mind to switch to `Array.every`? Reduce is less verbose than `.every`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

File: app/application/controller.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ export default Controller.extend({
   collections: [],
   session: service(),
   scmContexts: alias('model'),
-  showCollections: computed('collections.[]', function() {
+  showCollections: computed('collections.[]', function showCollections() {
```
Comment (adong) [inline app/application/controller.js]: My bad, it was a warning. Actually, I'm little confused why we have to give name to anonymous function. Maybe we can discuss on relaxing such rule.

File: app/commands/detail/controller.js (hunk 1 above)
Comment (DekusDenial) [reply app/commands/detail/controller.js]: my bad, should have used `Array.some`. Will make the change now

Comment (adong) [reply app/commands/detail/controller.js]: Yeah, let `babel` handles the transpile. 😃

File: app/application/controller.js (hunk 2 above)
Comment (DekusDenial) [reply app/application/controller.js]: i guess SD primarily bases its rule on airbnb. I dont mind relaxing it and dont think debugging will be harder without it

**Pull request merged**